deployer/src/main/java/com/percussion/services/pkginfo/impl/PSPkgInfoService.java

```diff
@@ -601,10 +611,11 @@ public List<PSPkgDependency> loadPkgDependenciesModifiable(IPSGuid guid,
       String temp = depType?"ownerPackageGuid":"dependentPackageGuid";
       Session session = getSession();
 
-         Criteria criteria = session.createCriteria(PSPkgDependency.class);
-         criteria.add(Restrictions.eq(temp, guid.longValue()));
-         pkgDeps = criteria.list();
-
+      CriteriaBuilder builder = getSession().getCriteriaBuilder();
+      CriteriaQuery<PSPkgDependency> criteria = builder.createQuery(PSPkgDependency.class);
+      Root<PSPkgDependency> critRoot = criteria.from(PSPkgDependency.class);
+      criteria.where(builder.equal(critRoot.get(temp),guid.longValue()));
+      pkgDeps = entityManager.createQuery(criteria).getResultList();
```

Comment:
return entityManager.createQuery(criteria).getResultList(); instead pf setting to pkgDeps and returning the variable.